editor/import/audio_stream_import_settings.cpp

```diff
@@ -540,14 +550,20 @@ AudioStreamImportSettingsDialog::AudioStreamImportSettingsDialog() {
 	loop->connect("toggled", callable_mp(this, &AudioStreamImportSettingsDialog::_settings_changed).unbind(1));
 	loop_hb->add_child(loop);
 	loop_hb->add_spacer();
-	loop_hb->add_child(memnew(Label(TTR("Offset:"))));
+	loop_offset_label = memnew(Label(TTR("Offset:")));
+	loop_hb->add_child(loop_offset_label);
 	loop_offset = memnew(SpinBox);
-	loop_offset->set_max(10000);
-	loop_offset->set_step(0.001);
-	loop_offset->set_suffix("sec");
+	loop_offset->set_custom_minimum_size(loop_offset->get_custom_minimum_size() + Size2(128, 0));
+	loop_offset->set_max(999999999);
+	loop_offset->set_step(0.00001);
 	loop_offset->set_tooltip_text(TTR("Loop offset (from beginning). Note that if BPM is set, this setting will be ignored."));
 	loop_offset->connect("value_changed", callable_mp(this, &AudioStreamImportSettingsDialog::_settings_changed).unbind(1));
 	loop_hb->add_child(loop_offset);
+	loop_use_samples = memnew(CheckBox);
+	loop_use_samples->set_text(TTR("Use Samples?"));
```

Comment:
```suggestion
	loop_use_samples->set_text(TTR("Use Samples"));
```
Other check boxes here don't use a question mark, nor do they seem to do so elsewhere